lib/progress_bar/enumerable.rb

```diff
@@ -0,0 +1,6 @@
+require_relative '../progress_bar'
```

Comment:
Lets rename this file to `lib/core_ext/enumerable_with_progress.rb`, to keep with the convention of putting monkeypatches into `core_ext`.